UI/forms/OBSBasicFilters.ui

```diff
@@ -414,7 +432,7 @@
          <item>
           <widget class="QDialogButtonBox" name="buttonBox">
            <property name="standardButtons">
-            <set>QDialogButtonBox::Reset|QDialogButtonBox::Close</set>
+            <set>QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
```

Comment:
Did these buttons get reordered for some reason?  Does this affect UI or tab order?

Reply:
QT designer reordered them. They still show up normally.